### PR TITLE
[strings] Fix and add polish translations, fix es-MX translations

### PR DESCRIPTION
**Pull request by lunarna-gh**
[old] This pull request fixes most unnatural-sounding polish strings, grammatical errors, and inaccuracies to their english counterparts. It also revamps the polish FAQ which was in a terrible state and adds several new polish strings.
I added myself to CONTRIBUTORS - I wasn't sure where to put myself, but I am willing to maintain the polish language so I'm currently under translation.
[new] New pull request to fix DCO issues and mexican spanish translations which were reverted to english by the first regeneration, possibly because someone edited the xml files without editing strings.txt. These match their `es` counterparts and I don't think they're wrong, though I don't speak spanish.

### CODE REVIEW

File: data/faq.html [hunk 1]
```diff
@@ -1341,18 +1340,18 @@ <h3 lang="tr">Uygulama konumumu haritada bulamıyor</h3>
...
+              urządzeniach nasza aplikacja będzie pokazywać przybliżoną
               lokalizację, o ile jesteś podłączony do Internetu.
             </p>
           </li>
 
           <li>
             <p>
-              Na koniec pamiętaj o lokalizacji określającej GPS (z WiFi i
-              telefonem komórkowym) Sieć wyłączona) może zająć trochę czasu. Im
+              Na koniec pamiętaj że określanie lokalizacji z GPS (z WiFi i
```
Comment (starsep) [inline data/faq.html:1350]: ``

Comment (starsep) [inline data/faq.html:1350]: przecinek przed że

File: data/strings/strings.txt [hunk 2]
```diff
@@ -12713,7 +12713,7 @@
     ko = 모든 지도 데이터는 OpenStreetMap 프로젝트에서 가져옵니다. ""장소를 추가하거나 편집할 수 있는 지도용 Wikipedia와 같으며 전 세계 수백만 사용자가 사용할 수 있습니다.
     nb = Alle kartdata kommer fra OpenStreetMap-prosjektet. Det er som Wikipedia for kart hvor du kan legge til eller redigere steder, og de blir tilgjengelige for millioner av brukere over hele verden.
     nl = Alle kaartgegevens komen uit het OpenStreetMap-project. Het is als Wikipedia voor kaarten waar je plaatsen kunt toevoegen of bewerken, en ze worden beschikbaar voor miljoenen gebruikers over de hele wereld.
-    pl = Wszystkie dane mapy pochodzą z projektu OpenStreetMap. Przypomina Wikipedię z mapami, w których można dodawać lub edytować miejsca, a stają się one dostępne dla milionów użytkowników na całym świecie.
+    pl = Wszystkie dane mapy pochodzą z projektu OpenStreetMap. Jest jak Wikipedia, tylko dla map. Możesz dodawać lub edytować miejsca i staną się dostępne milionom użytkowników na całym świecie.
```
Comment (starsep) [inline data/strings/strings.txt:12716]: `dla milionów` brzmiało poprawnie

File: data/strings/strings.txt [hunk 3]
```diff
@@ -24034,7 +24034,7 @@
     mr = चालू केल्यास निष्क्रियतेच्या कालावधीनंतर स्क्रीनला झोपण्याची परवानगी दिली जाईल.
     nb = Når den er aktivert, får skjermen lov til å sove etter en periode med inaktivitet.
     nl = Indien ingeschakeld, mag het scherm slapen na een periode van inactiviteit.
-    pl = Po włączeniu ekran będzie mógł spać po okresie bezczynności.
+    pl = Po włączeniu tej funkcji ekran będzie będzię mógł się wygasić po okresie bezczynności.
```
Comment (starsep) [inline data/strings/strings.txt:24037]: do usunięcia `będzię`

**Review by starsep — CHANGES_REQUESTED** [review]

**Review by biodranik — CHANGES_REQUESTED** [review]
Thanks! Please next time do not close/reopen the Pull Request, but try to rebase your code and fix merge conflicts if necessary. It is usually easily done via interactive rebase, where you remove your regenerated commits, then finish rebase, and then regenerate them again.

Also, please combine two regenerated commits into the one, so you'll have:
1. Polish changes
2. es-MX
3. regenerated.

You can easily do it with an interactive rebase. If you never did that, just make a copy of your branch and experiment first.

**Comment by lunarna-gh** [comment]
Everything, including the errors pointed out by @starsep, should be fixed now. Thanks for telling me about closing PRs and rebasing, will remember to do it this way in the future.

**Review by starsep — APPROVED** [review]

**Review by biodranik — APPROVED** [review]
Thank you for the contribution! Welcome to the club )

**Pull request merged**